### ISSUE TITLE
physical system description missing figure

### ISSUE THREAD

**Issue by salahhessien** [JohnErnsthausen/roc#5]
@JohnErnsthausen 
I think it would be useful if you could include a diagram in the physical system description section to help visualizing and understanding the element of your solver.

**Comment by smiths** [comment]
@JohnErnsthausen, if you add a figure, please include the commit hash that adds the figure as part of the comment when closing the issue.

**Comment by JohnErnsthausen** [comment]
Will do. What picture might help for a schematic @salahhessien? Is it the sequence, the process, or the output that needs clarification?

**Comment by salahhessien** [comment]
@JohnErnsthausen if I were you, I would summarize the algorithm sequence of operations in a flow chart or so.

**Comment by JohnErnsthausen** [comment]
This issue was subsequently raise by @smiths in my SRS feedback. The request was

P2 Show show how ROC fits into a typical context

**Comment by JohnErnsthausen** [comment]
@smiths and @salahhessien,

I added Figure 1: System Context

[commit a57b80](a57b80f051026b0643b6f4acdddebc90613d09b3)

I did not add a flow diagram. One major constraint is that ROC is a research project and I do not know what the flow diagram will be. At this point, I plan on analyzing the results of Real Pole, Complex Pair Poles, and Top Line Analysis over many use cases to understand the capability of resolving truncated sequences. 

Thank you for this issue!
